### PR TITLE
[test] add e2e test for linux communication through clusterIP service

### PR DESCRIPTION
**Pull request by JacobTanenbaum**
Adding an e2e test to test linux communication through a service

JIRA: https://issues.redhat.com/browse/WINC-496

### CODE REVIEW

File: test/e2e/network_test.go [hunk 1]
```diff
@@ -70,6 +70,17 @@ func testEastWestNetworking(t *testing.T) {
 		err = testCtx.waitUntilJobSucceeds(linuxCurlerJob.Name)
 		assert.NoError(t, err, "could not curl the Windows server from a linux container")
 
+		//Create a ClusterIP service
+		clusterIPSVC, err := testCtx.createService(winServerDeployment.Name, v1.ServiceTypeLoadBalancer, *winServerDeployment.Spec.Selector)
```
Comment (LorbusChris) [inline test/e2e/network_test.go:74]: shouldn't this be `v1.ServiceTypeClusterIP`?

File: test/e2e/network_test.go [hunk 2]
```diff
@@ -70,6 +70,17 @@ func testEastWestNetworking(t *testing.T) {
 		err = testCtx.waitUntilJobSucceeds(linuxCurlerJob.Name)
 		assert.NoError(t, err, "could not curl the Windows server from a linux container")
 
+		//Create a ClusterIP service
```
Comment (aravindhp) [inline test/e2e/network_test.go:73]: Missing space after `//`

I suggest that we separate this out into sub-tests for each service or even make it [table driven](https://github.com/golang/go/wiki/TableDrivenTests).

**Review by aravindhp — CHANGES_REQUESTED** [review]
Thanks for working on this, @JacobTanenbaum. Please address my comments.

File: test/e2e/network_test.go (hunk 1 above)
Comment (JacobTanenbaum) [reply test/e2e/network_test.go]: Good catch, fixed

File: test/e2e/network_test.go (hunk 2 above)
Comment (JacobTanenbaum) [reply test/e2e/network_test.go]: Put in the space. Do you want me to work on separating the tests here or do you guys want to add it to your next sprint?

Comment (aravindhp) [reply test/e2e/network_test.go]: That is up to you. If you don't have cycles, someone can take over this PR from you during the next sprint.

**Comment by aravindhp** [comment]
Superseded by #129

**Pull request closed without merging**